gallery/src/components/NxTabs/NxTabsPage.tsx

```diff
@@ -0,0 +1,102 @@
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow
+} from '@sonatype/react-shared-components';
+
+import NxTabsSimpleExample from './NxTabsSimpleExample';
+
+const NxTabsSimpleExampleCode = require('!!raw-loader!./NxTabsSimpleExample').default;
+
+export default function NxTabsPage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <p className="nx-p">
+          A set of accessible tabs components which must be used together.
+        </p>
+
+        <section className="nx-tile--subsection">
+          <h3 className="nx-h3">NxTabs</h3>
+
+          <p className="nx-p">
+            The top-level container for tabbed navigation.
+            It can have <code className="nx-code">&lt;NxTabList&gt;</code> and
+            {' '}<code className="nx-code">&lt;NxTabPanel&gt;</code> components as children.
+          </p>
+
+          <NxTable className="nx-table--gallery-props">
+            <NxTableHead>
+              <NxTableRow>
+                <NxTableCell>Prop</NxTableCell>
+                <NxTableCell>Type</NxTableCell>
+                <NxTableCell>Details</NxTableCell>
+              </NxTableRow>
+            </NxTableHead>
+            <NxTableBody>
+              <NxTableRow>
+                <NxTableCell>activeTab</NxTableCell>
+                <NxTableCell>number</NxTableCell>
+                <NxTableCell>
+                  The index of the active tab.
+                </NxTableCell>
+              </NxTableRow>
+              <NxTableRow>
+                <NxTableCell>onTabSelect</NxTableCell>
+                <NxTableCell>function(number)</NxTableCell>
+                <NxTableCell>
+                  Called with the index of the newly selected tab when the currently selected tab changes.
+                </NxTableCell>
+              </NxTableRow>
+            </NxTableBody>
+          </NxTable>
+        </section>
+
+        <section className="nx-tile--subsection">
+          <h3 className="nx-h3">NxTabList</h3>
+
+          <p className="nx-p">
+            The parent container for the <code className="nx-code">&lt;NxTab&gt;</code> components.
+          </p>
+        </section>
+
+        <section className="nx-tile--subsection">
+          <h3 className="nx-h3">NxTab</h3>
+
+          <p className="nx-p">
+            The component the user selects to switch tabs.
+            The index prop is automatically configured by the <code className="nx-code">NxTabs</code> component.
+            There should be one of these for each <code className="nx-code">NxTabPanel</code> component.
```

Comment:
Mention that it takes all attrs that can go in a `<li>`

Reply:
ed7e704

Reply:
2ad17fa5e16800c8e1f34c7ea4d027936c2e24a4